[2022-12-04 13:29:51,831] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: coba.visual manual__2022-12-04T06:29:28.843827+00:00 [queued]>
[2022-12-04 13:29:51,834] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: coba.visual manual__2022-12-04T06:29:28.843827+00:00 [queued]>
[2022-12-04 13:29:51,834] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-12-04 13:29:51,835] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-12-04 13:29:51,835] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-12-04 13:29:51,953] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): visual> on 2022-12-04 06:29:28.843827+00:00
[2022-12-04 13:29:51,959] {standard_task_runner.py:52} INFO - Started process 7101 to run task
[2022-12-04 13:29:51,962] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'coba', 'visual', 'manual__2022-12-04T06:29:28.843827+00:00', '--job-id', '74', '--raw', '--subdir', 'DAGS_FOLDER/AirFlowScheaduling.py', '--cfg-path', '/tmp/tmpmxx9ruin', '--error-file', '/tmp/tmpozojs1lb']
[2022-12-04 13:29:51,962] {standard_task_runner.py:80} INFO - Job 74: Subtask visual
[2022-12-04 13:29:51,999] {task_command.py:371} INFO - Running <TaskInstance: coba.visual manual__2022-12-04T06:29:28.843827+00:00 [running]> on host UbuntuPC
[2022-12-04 13:29:52,033] {taskinstance.py:1589} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Kelompok6
AIRFLOW_CTX_DAG_ID=coba
AIRFLOW_CTX_TASK_ID=visual
AIRFLOW_CTX_EXECUTION_DATE=2022-12-04T06:29:28.843827+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-12-04T06:29:28.843827+00:00
[2022-12-04 13:29:52,399] {warnings.py:109} WARNING - /home/noname00/.local/lib/python3.8/site-packages/pandas/plotting/_matplotlib/tools.py:227: UserWarning: Starting a Matplotlib GUI outside of the main thread will likely fail.
  fig = plt.figure(**fig_kw)

[2022-12-04 13:29:52,512] {warnings.py:109} WARNING - /home/noname00/.local/lib/python3.8/site-packages/matplotlib/backends/_backend_gtk.py:141: DeprecationWarning: Gtk.Window.set_wmclass is deprecated
  self.window.set_wmclass("matplotlib", "Matplotlib")

[2022-12-04 13:29:53,981] {python.py:173} INFO - Done. Returned value was: None
[2022-12-04 13:29:53,985] {taskinstance.py:1415} INFO - Marking task as SUCCESS. dag_id=coba, task_id=visual, execution_date=20221204T062928, start_date=20221204T062951, end_date=20221204T062953
[2022-12-04 13:29:54,093] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-04 13:29:54,115] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
